source/includes/_evidence.md

```diff
@@ -325,30 +325,8 @@ curl https://api.gresb.com/api/v0/entities/{entity_id}/responses/{response_id}/v
 ```
 
 ## Evidence Delete
-Evidence documents can be removed at once with a `DELETE` request which will set the evidence type, URL, notes, and sharing settings to `null` in the corresponding endpoints. 
-
-### DELETE /documents/{item_number}
```

Comment:
Removed this whole part to have everyone use the most convenient way instead of the elaborate way?